src/components/ThemeProvider.js

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
 import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import injectTapEventPlugin from 'react-tap-event-plugin';
 
 import Theme from './Theme';
 
+injectTapEventPlugin();
```

Comment:
I thought that adding it here we could remove on the apps. I'll remove from here... 😄